indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -581,6 +649,9 @@ protected void reduce(
             toMerge.add(finalFile);
           }
 
+          Futures.allAsList(persistFutures).get(1, TimeUnit.HOURS);
```

Comment:
do we need to hardcode a timeout here? wouldn't hadoop automatically kill the containers if no progress is being made.